helpers/html.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+require 'sinatra/base'
+
+module Sinatra
+  module HTMLEscapeHelper
+    def h(text)
+      Rack::Utils.escape_html(text)
+    end
+  end
+
+  helpers HTMLEscapeHelper
```

Comment:
With the `PopoloHelper` we [set it up](https://github.com/everypolitician/viewer-sinatra/blob/ee9d1dd664f4094a85e9ba7b38338213fb14b570/app.rb#L18) in `app.rb` itself. I'm not sure whether it's generally preferred to do it that way or this, but again I think we should be consistent.